src/plants/PlantSelectField.js

```diff
@@ -105,75 +116,58 @@ class PlantSelectField extends React.Component {
     const { searchTerm, showResults, isLoading } = this.state;
 
     return (
-      <ClickAwayListener onClickAway={this.closeResults}>
-        <div className={classes.selectWraper}>
-          {selectedPlant && (
-            <List component={Paper}>
-              <PlantItem
-                plant={selectedPlant}
-                classes={classes}
-                deselectPlant={this.deselectPlant}
-              />
-            </List>
+      <>
+        <Autocomplete
+          value={selectedPlant}
+          onChange={(e, newSelectedPlant) => this.selectPlant(newSelectedPlant)}
+          inputValue={searchTerm}
+          onInputChange={(e, newInputValue) =>
+            this.setState((state) => ({ ...state, searchTerm: newInputValue }))
+          }
+          open={showResults}
+          onOpen={() => this.openResults()}
```

Comment:
```suggestion
          onOpen={this.openResults}
```
já que essa função já esta "bind" não precisa de criar outra função pra chamar ela

na linha de baixo onde esta o `onClose` mesma coisa

Reply:
o bind me dá umas rasteiras de vez em quando hahaha mas tá arrumado já :+1: 

valeu pelo CR!